package.json

```diff
@@ -99,19 +99,19 @@
     "@sentry/react": "6.16.1",
     "@sentry/tracing": "6.16.1",
     "@stacks/blockchain-api-client": "0.65.0",
-    "@stacks/common": "4.0.1",
-    "@stacks/connect": "6.7.0",
+    "@stacks/common": "4.2.1",
+    "@stacks/connect": "https://github.com/beguene/connect-tgz/raw/main/stacks-connect-v6.8.4.tgz",
```

Comment:
This version has been released ...same with `connect-react` below.

Reply:
This is not the same as the released v6.8.4. This packages is actually https://github.com/hirosystems/connect/pull/236. I will update it when we release my connect PR.
Same for connect-react.